package.json

```diff
@@ -24,7 +24,10 @@
     "test": "mocha --recursive 'test/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
     "test:watch": "npm run test -- -w ./lib",
     "lint": "jshint lib/ --config .jshintrc && jshint test/ --config test/.jshintrc",
+    "lint:md": "remark 'README.md' documentation",
```

Comment:
The three new targets should be added to https://github.com/telefonicaid/perseo-fe/blob/master/documentationT/development.md#development-documentation

In addition, what the diference between `remark` and `textlint`? Could you provide some link about it?

Reply:
The links were in the head of the PR:

* [remark-lint](https://github.com/remarkjs/remark-lint) - check that the Markdown is well formated.
* [textlint](https://github.com/textlint/textlint) - check for spelling mistakes, dead links, poor grammar.

## remark lint

`remark-lint` is required because `prettier` can only do so much with random human markup particularly if you've mixed up `*` and `-` for bullets and haven't spaced the bullets properly or used inconsistent bolding.

How can you tell how this should **really be rendered?
```
* **XXXXX*- YYYY
```

`remark-lint`  has the same role for **markdown** as `eslint` does for **JavaScript** - it means you can be confident that the prettier result is good.

## text lint

This does nothing to the markup - it merely checks for consistent text and links so words like FIWARE or IoT Agent can be enforced to have consistency (which is good for documentation). It also adds a link checker so the docs shouldn't contain any dead links.

Reply:
Targets added to documentation - fixed b4923d7